src/core/storage-manager.ts

```
@@ -0,0 +1,85 @@
+  }
+
+  /**
+   * Close storage connection
+   */
+  static async close(): Promise<void> {
+    if (StorageManager.instance) {
+      await StorageManager.instance.close();
+      StorageManager.instance = null;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)